include/clang/Interpreter/CppInterOp.h

```diff
@@ -536,7 +543,7 @@ namespace Cpp {
   };
   /// Builds a template instantiation for a given templated declaration.
   CPPINTEROP_API TCppScope_t InstantiateTemplate(TCppScope_t tmpl,
```

Comment:
We need to document the parameters, too.

Reply:
done